packages/components/scss/bootstrap_override_functions.scss

```diff
@@ -0,0 +1,36 @@
+// Bootstrap overrides for `bootstrap/scss/_functions.scss`
+
+// Color contrast
+// DH Customization: Add support for dh-color variables
+@function color-yiq(
+  $color,
+  $dark: var(--dh-color-gray-50),
+  $light: var(--dh-color-gray-900)
+) {
+  // if $color is a hex color, use the default color-yiq function
+  @if type-of($color) == 'color' {
+    $r: red($color);
+    $g: green($color);
+    $b: blue($color);
+
+    $yiq: (($r * 299) + ($g * 587) + ($b * 114)) * 0.001;
+
+    @if $yiq >= $yiq-contrasted-threshold {
+      @return $dark;
+    } @else {
+      @return $light;
+    }
+  }
+
+  // if the $color contains the string "yellow" or "orange" or "chatreuse" or "cyan" return $dark
+  // per the spectrum color guidelines. All other colors have white text.
+  // https://spectrum.adobe.com/page/using-color/#Solid-color-background-with-black-or-white
+  // Which for us means, informative and notice have dark text.
+  @if str-index($color, 'informative') {
```

Comment:
Note to myself, I need to update this since we now use our own --dh-color vars instead of Spectrum variables